releases/23.0.0.6/full/Dockerfile.ubuntu.openjdk11

```diff
@@ -18,28 +43,52 @@ LABEL org.opencontainers.image.authors="Arthur De Magalhaes, Chris Potter, Leo C
       org.opencontainers.image.title="Open Liberty" \
       org.opencontainers.image.version="$LIBERTY_VERSION"
 
+COPY NOTICES /opt/ol/NOTICES
 COPY helpers /opt/ol/helpers
 COPY fixes/ /opt/ol/fixes/
 
-# Install Open Liberty
-RUN apt-get update \
-    && apt-get install -y --no-install-recommends unzip openssl wget \
-    && rm -rf /var/lib/apt/lists/* \
-    && wget -q $LIBERTY_DOWNLOAD_URL -U UA-Open-Liberty-Docker -O /tmp/wlp.zip \
-    && echo "$LIBERTY_SHA  /tmp/wlp.zip" > /tmp/wlp.zip.sha1 \
-    && sha1sum -c /tmp/wlp.zip.sha1 \
-    && unzip -q /tmp/wlp.zip -d /opt/ol \
-    && rm /tmp/wlp.zip \
-    && rm /tmp/wlp.zip.sha1 \
-    && mkdir -p /licenses \
-    && cp /opt/ol/wlp/LICENSE /licenses/ \
-    && apt-get remove -y unzip \
-    && apt-get remove -y wget \
-    && rm -rf /var/lib/apt/lists/* \
-    && useradd -u 1001 -r -g 0 -s /usr/sbin/nologin default \
+# Install dumb-init
+RUN set -eux; \
+    ARCH="$(dpkg --print-architecture)"; \
+    case "${ARCH}" in \
+       aarch64|arm64) \
+         DUMB_INIT_URL='https://github.com/Yelp/dumb-init/releases/download/v1.2.5/dumb-init_1.2.5_aarch64'; \
+         DUMB_INIT_SHA256=b7d648f97154a99c539b63c55979cd29f005f88430fb383007fe3458340b795e; \
+         ;; \
+       amd64|x86_64) \
+         DUMB_INIT_URL='https://github.com/Yelp/dumb-init/releases/download/v1.2.5/dumb-init_1.2.5_x86_64'; \
+         DUMB_INIT_SHA256=e874b55f3279ca41415d290c512a7ba9d08f98041b28ae7c2acb19a545f1c4df; \
+         ;; \
+       ppc64el|ppc64le) \
+         DUMB_INIT_URL='https://github.com/Yelp/dumb-init/releases/download/v1.2.5/dumb-init_1.2.5_ppc64le'; \
+         DUMB_INIT_SHA256=3d15e80e29f0f4fa1fc686b00613a2220bc37e83a35283d4b4cca1fbd0a5609f; \
+         ;; \
+       s390x) \
+         DUMB_INIT_URL='https://github.com/Yelp/dumb-init/releases/download/v1.2.5/dumb-init_1.2.5_s390x'; \
+         DUMB_INIT_SHA256=47e4601b152fc6dcb1891e66c30ecc62a2939fd7ffd1515a7c30f281cfec53b7; \
+         ;;\
+       *) \
+         echo "Unsupported arch: ${ARCH}"; \
+         exit 1; \
+         ;; \
+    esac; \
+    curl -LfsSo /usr/bin/dumb-init ${DUMB_INIT_URL}; \
+    echo "${DUMB_INIT_SHA256} */usr/bin/dumb-init" | sha256sum -c -; \
+    chmod +x /usr/bin/dumb-init;
+
+# Add default user 1001 and create wlp with right user/permissions before copying
+RUN adduser -u 1001 -r -g root -s /usr/sbin/nologin default \
```

Comment:
Ubuntu-based files had this instead:
`useradd -u 1001 -r -g 0 -s /usr/sbin/nologin default \`

Reply:
hmm, I thought `adduser` is preferred over `useradd`?

Reply:
Possibly, `adduser` is a wrapper for `useradd`. The first stage uses `useradd` while the second stage uses `adduser`. We should use the same one. I am not sure if we even need it (plus chown and chmod) in the first stage